src/main/java/icbm/classic/mods/cubicchunks/CubicChunks.java

```diff
@@ -0,0 +1,60 @@
+package icbm.classic.mods.cubicchunks;
+
+import icbm.classic.ICBMClassic;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.Loader;
+
+import java.lang.reflect.Method;
+
+public class CubicChunks {
+
+    public static CubicChunks INSTANCE = new CubicChunks();
+    boolean present = false;
+
+    Method getMinHeightMethod;
+    Method getMaxHeightMethod;
+
+    @SuppressWarnings("JavaReflectionMemberAccess")
+    private CubicChunks() {
+        ICBMClassic.logger().info("ICBMClassic>>CubicChunks: Looking for CubicChunks");
+        if (!Loader.isModLoaded("cubicchunks")) {
+            ICBMClassic.logger().info("ICBMClassic>>CubicChunks: Didn't find CubicChunks");
+            return;
```

Comment:
I rather see the if wrapped around the whole block rather than an early `return`